### PR TITLE
polymc: Update to 1.4.0, add polymc-qt5

### PR DESCRIPTION
**Pull request by DioEgizio**
## What package release type is this?

> - [x] stable
> - [ ] beta
> - [ ] dev/nightly/canary

### CODE REVIEW

File: bucket/polymc.json [hunk 1]
```diff
@@ -24,13 +16,6 @@
...
-                "url": "https://github.com/PolyMC/PolyMC/releases/download/$version/PolyMC-Windows-x86_64-$version.zip"
-            },
-            "32bit": {
-                "url": "https://github.com/PolyMC/PolyMC/releases/download/$version/PolyMC-Windows-i686-$version.zip"
-            }
-        }
+        "url": "https://github.com/PolyMC/PolyMC/releases/download/$version/PolyMC-Windows-$version.zip"
     }
-}
+}
```
Comment (Scrumplex) [inline bucket/polymc.json:21]:
```suggestion
}

```

File: bucket/polymc-qt5.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+            "PolyMC"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/PolyMC/PolyMC"
+    },
+    "autoupdate": {
+        "url": "https://github.com/PolyMC/PolyMC/releases/download/$version/PolyMC-Windows-Legacy-$version.zip"
+    }
+}
```
Comment (Scrumplex) [inline bucket/polymc-qt5.json:21]:
```suggestion
}

```

**Comment by Calinou** [comment]
Should the qt5 manifest be renamed to `polymc-legacy` to match the official download name?

**Comment by DioEgizio** [comment]
idk, polymc legacy feels like it's the legacy version of PolyMC but the legacy there is referred to the windows version

**Comment by Scrumplex** [comment]
Are there other cases where there is a build for older Windows versions? IMO neither `-qt5` nor `-legacy` are good names

**Review by Calinou — APPROVED** [review]
Thanks!

Will merge this for now so that PolyMC is up-to-date. Feel free to discuss the `polymc-qt5` manifest naming in a separate issue.

**Pull request merged**